frontend/src/Context/PublisherProvider/usePublisher/usePublisher.spec.tsx

```diff
@@ -20,13 +20,15 @@ const defaultSettings = {
   publishAudio: false,
   publishVideo: false,
   name: '',
-  blur: false,
   noiseSuppression: true,
+  publishCaptions: false,
 };
 const mockUserContextWithDefaultSettings = {
   user: {
     defaultSettings,
+    issues: { reconnections: 0, audioFallbacks: 0 },
```

Comment:
did this need to be added because something was breaking? if it's not related to background blur, let's fix it in a separate PR 🙏